content/blog/D0014-openage_mod_api_finale_0.1.md

```diff
@@ -0,0 +1,79 @@
+Title: D14: Openage modding API - Finale
+Date: 2019-05-31
+Tags: nyan, modding, API
+Authors: heinezen
+Summary: We look back at our API changes and summarize the new features
+
+Over the last year, we have introduced a lot of features to our API and spent a lot of time polishing and improving its core concepts. But as an old Chinese saying goes: "You have to let go, otherwise you will never finish the converter and jj will be unhappy". So today, we finally present you the first release version of the *openage Modding API* tree and will talk a little bit about the features that come with it.
+
+![modding API tree v0.1a]({static}/images/D0014-API-tree-0.1.svg)
```

Comment:
i think we should add something like "we present you the result of hours of thought and discussions:" as a introduction to this image.